src/JS/taskOperations.js

```diff
@@ -0,0 +1,92 @@
+let tasksList = [];
+
+const addOnLocalStorage = (task) => {
+  tasksList = JSON.parse(localStorage.getItem('tasks: '));
+  if (tasksList == null) tasksList = [];
+  tasksList.push(task);
+  localStorage.setItem('tasks: ', JSON.stringify(tasksList));
+};
+
+const addTaskToPage = (taskToBeDone) => {
+  const tasksContainerElt = document.querySelector('.tasks-container');
+  const taskDiv = document.createElement('div');
+  const taskContentDiv = document.createElement('div');
+  const taskValue = document.createElement('input');
+  const checkboxElt = document.createElement('input');
+  const trashIcon = document.createElement('i');
+  const dotsIcon = document.createElement('i');
+  taskDiv.setAttribute('class', 'task');
+  taskContentDiv.setAttribute('class', 'task-content');
+  taskValue.setAttribute('class', 'task-value');
+  taskValue.setAttribute('type', 'text');
+  checkboxElt.setAttribute('type', 'checkbox');
+  trashIcon.setAttribute('class', 'fa fa-trash trash-icon');
+  dotsIcon.setAttribute('class', 'fa fa-ellipsis-v dots-icon');
+  checkboxElt.checked = taskToBeDone.completed;
+  taskValue.value = taskToBeDone.description;
+  taskContentDiv.appendChild(checkboxElt);
+  taskContentDiv.appendChild(taskValue);
+  taskDiv.appendChild(taskContentDiv);
+  taskDiv.appendChild(trashIcon);
+  taskDiv.appendChild(dotsIcon);
+  tasksContainerElt.appendChild(taskDiv);
+};
+
+const loadTasksFromLS = () => {
+  tasksList = JSON.parse(localStorage.getItem('tasks: '));
+  if (tasksList !== null) {
+    tasksList.forEach((taskToBeDone) => {
+      addTaskToPage(taskToBeDone);
+    });
+  } else {
+    tasksList = [];
+  }
+};
+
+const addTaskToArray = (taskToBeDone) => {
+  const task = {
+    description: String,
+    completed: false,
+    index: Number,
+  };
+  task.description = taskToBeDone;
+  task.index = tasksList.length;
+  addOnLocalStorage(task);
+  addTaskToPage(task);
+};
+
+const removeTask = (index) => {
+  const tasksElt = document.querySelectorAll('.task');
+  tasksList.splice(index, 1);
+  tasksElt[index].remove();
+  for (let i = index + 1; i < tasksList.length; i += 1) {
+    tasksList[i].index -= 1;
+  }
+  localStorage.setItem('tasks: ', JSON.stringify(tasksList));
+};
```

Comment:
- [ ] After removing a todo the index number of each todo should reset. This means if you have 5 todos and if you remove 3rd todo then your fourth and fifth todo index number should decrease by 1. I can see you have implemented it but it is not working. Please fix it :+1: